src/client/components/personalization/productCard.module.scss

```
@@ -0,0 +1,103 @@
+/**
+* Product Card
+*/
+.productCard {
+  border-radius: 8px;
+  padding: rem(16px);
+  border: 1px solid v('gray-box-stroke');
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(245px, 1fr));
+  gap: rem(24px);
+
+  @include media('<=phone') {
+    grid-template-columns: 1fr;
+    padding: rem(12px);
+  }
+}
+
+.productCardImage {
+  max-width: 100%;
+  object-fit: cover;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.productCardBody {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.productCardName {
+  color: v('dark-black');
+  font-size: rem(24px);
+  font-weight: 500;
+  line-height: 130%;
+
+  @include media('<=phoneLandscape') {
+    font-size: rem(20px);
+  }
+}
+
+.productCardSize {
+  color: v('dark-black');
+  font-size: rem(16px);
+  font-weight: 300;
+  line-height: 145%;
+}
+
+.productCardBodyBottomRow {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  gap: rem(8px);
+}
+
+.productCardPrice {
+  color: v('orange-gradient');
+  font-size: rem(16px);
+  line-height: 160%;
+  letter-spacing: 0.16px;
+}
+
+.productCardButtons {
+  display: flex;
+  gap: rem(8px);
+  flex-wrap: wrap;
+}
+
+.addToFavorites {
+  padding: 8px !important;
+}
```

💩 
Let's fix it in other way?

```suggestion
.productCardButtons  .addToFavorites {
  padding: 8px;
}
```

---

Missed that, thank you!